packages/radio-group/src/Radio/Radio.tsx

```diff
@@ -75,6 +77,9 @@ function Radio({
             [css`
               font-size: 12px;
             `]: size === Size.XSmall, // TODO: keeping this style until XS is deprecated
+            [css`
```

Comment:
nit: feels more legible to say the bold font-weight should be applied when the `bold === true` than to negate

Reply:
On one hand I agree, but this way  implies that `bold === true` is the default